tests/run/Makefile

```diff
@@ -10,9 +10,9 @@ SOURCES = $(SUBOBJDIR)/run_u.c host.c
 
 INCLUDES = $(OEHOST_INCLUDES) -I$(SUBOBJDIR)
 
-CFLAGS = $(OEHOST_CFLAGS)
+CFLAGS = $(OEHOST_CFLAGS) $(GCOV_CFLAGS)
```

Comment:
Should protect $(GCOV_CFLAGS) with ifdef MYST_ENABLE_GCOV

Reply:
Added this

Reply:
Actually, I think tests/run belongs to a test framework that we no longer used. Can we remove the folder?